build/build-undetermined.js

```diff
@@ -0,0 +1,54 @@
+/*!***************************************************
+ * diacritics
+ * http://diacritics.io/
+ * Copyright (c) 2016–2018 The Diacritics Authors
+ * Released under the MIT license https://git.io/vXg2H
+ *****************************************************/
+'use strict';
+const fs = require('fs'),
+  und = require('./templates/und.json'),
+
+  // This file only needs to be run once, or if the diacritics list is updated!
+  // copied from https://git.io/vNPua
+  diacritics = [
+    'aàáảãạăằắẳẵặâầấẩẫậäåāą', 'AÀÁẢÃẠĂẰẮẲẴẶÂẦẤẨẪẬÄÅĀĄ',
```

Comment:
What will happen if we have diacritics that are included in the "und" and another language? For example I guess since you've added the PL language to the validated list, this diacritics list contains them too.

Reply:
We have characters defined in multiple languages already. If you want to build the array for mark.js, then only target the `und` language.

Reply:
So they are generated redundant?
Using `und` isn't future proof. For me this is a temporary workaround until we've classified the diacritics and created the corresponding language files. Then, `und` will not be completed or empty at all.

Reply:
@Mottie So, if I would roll out mark.js with the "und" language I could not only use this undetermined language since I can not "pin" the output to a specific "und" state so once the diacritics inside there would be converted to a different language I'd loose them. Our versioned database & API only covers schema changes, not data changes, which I like since usually this means that you automatically receive updates.

So I'd need to use all languages. The question then is, if they are generated redundant? So I created an entry in "de.json" for "à" and tested it. It seems to be redundantly generated. So for the "und" case in my opinion it's necessary to check if the diacritic is included in another language and if so, ignore it and print a warning to the console. This should be part of the generation process of the entire build, not the generation process of und.json since this may only be important in future once other guys create language files.

What do you think? If you agree, can you take over this part or should I?